starlette_exporter/middleware.py

```diff
@@ -40,7 +40,7 @@ def get_matching_route_path(
             # be the complete path (it may represent the path to the
             # mounted router). If this is a mounted route, descend into it to
             # get the complete path.
-            if isinstance(route, Mount) and route.routes:
+            if hasattr(route, "routes"):
```

Comment:
another option here is to have
```suggestion
if isinstance(route, BaseRoute) and route.routes:
```